content/en/tracing/service_catalog/scorecards.md

```diff
@@ -0,0 +1,88 @@
+
+### Production readiness
+
+The production readiness score for all services (unless otherwise indicated) is based on these rules:
+
+Have any SLOs defined 
+: [Service Level Objectives (SLOs)][2] provide a framework for defining clear targets around application performance, which helps you provide a consistent customer experience, balance feature development with platform stability, and improve communication with internal and external users.
+
+Have any monitors defined
+: Monitors reduce downtime by helping your team quickly react to issues in your environment. Review [recommended monitors][3].
+
+Specified on-call
+: Improve the on-call experience for everyone by establishing clear ownership of your services. This will give your on-call engineers the correct point of contact during incidents, reducing the time it takes to resolve your incidents. 
+
+Last deployment occurred within the last 3 months
+: For services monitored by APM or USM.	Agile development practices give you the ability to quickly address user feedback and pivot to developing the most important functionality for your end users. 
+
+### Observability best practices
+
+The Observability best practices score is based on the following rules:
+
+Deployment tracking is active
+: For services monitored by APM or USM. [Ensure smooth rollouts by implementing a version tag][4]. As you roll out new versions of your functionality, Datadog captures and alerts on differences between the versions in error rates, number of requests, and more. This can help you understand when to roll back to previous versions to improve end user experience. 
+
+Logs correlation is active
+: For APM services. [Correlation between APM and Logs][5] improves the speed of troubleshooting for end users, saving you time during incidents and outages. 
+
+### Ownership and documentation
+
+The Ownership and documentation score is based on the following rules:
+
+Team defined
+: Defining a Team makes it easier for your on-call staff to know which team to escalate to in case a service they are not familiar with is the root cause of an issue. 
+
+Contacts defined
+: Defining contacts reduces the time it takes for your on-call staff to escalate to the owner of another service, helping you recover your services faster from outages and incidents.
+
+Code repos defined
+: Identifying code repositories enables your engineers to perform an initial investigation into an issue without having to contact the service's owning team. This improves collaboration and helps your engineers increase their overall understanding of integration points.
+
+Any docs defined
+: In the Service Catalog Other Links section, specify additional links to resources such as runbooks, dashboards, or other internal documentation. This helps with initial investigations and provides quick access to emergency remediation runbooks for outages and incidents.
```

Comment:
I'm not exactly sure where this section is located, but is fine for now~

Reply:
Yeah, when this is less beta-y, we'll link it to the instructions about Setting Other Links